workload/scripts/workload-install.sh

```diff
@@ -61,6 +61,19 @@ function read_dotnet_link() {
     echo $PWD
 }
 
+function test_directory() {
```

Comment:
How about ... 

```bash
function error_permission_denied() {
    echo "No permission to install manifest. Try again with sudo."
    exit 1
}

function ensure_directory() {
    if [ ! -d $1 ]; then
        [ -w dirname("$1") ] && mkdir -p $1 || error_permission_denied
    fi
    [ ! -w $1 ] || error_permission_denied
}
```

Reply:
`[ -w dirname("$1") ]`  occurred problem while creating a directory with a depth of 2 or more.